lib/utils/utilities.js

```diff
@@ -40,6 +40,57 @@ function getSimulatorsFolder() {
     return simulatorsFolderPath;
 }
 
+function getDeviceId() {
```

Comment:
Can this perhaps be renamed to something like `getSimulatorModel`? The name sort of implies that it gets the id for a physical device (when compared to `getSimulatorId`)